samples/bluetooth/central_gatt_write/src/gatt_write_common.c

```diff
@@ -17,6 +17,10 @@
 static uint32_t phy_update_countdown;
 static uint8_t phy_param_idx;
 
+/* Count down number of metrics intervals before performing a param update */
+#define PARAM_UPDATE_COUNTDOWN PHY_UPDATE_COUNTDOWN
```

Comment:
Also, this feels like even more test code that isn't related to GATT :( Would prefer that we test things like this in BSIM, instead of polluting a GATT sample with PHY tests (personal opinion)

Reply:
This may not be related to GATT but a typical user application requiring a GATT Write without Response will perform both PHY update and Connection Update to tune the throughput. In that sense, is a sample that demonstrate GATT Write without Response, PHY Update and Connection Update (there are no samples for these procedures currently in the upstream repo!).